style.css

```diff
@@ -0,0 +1,90 @@
+div h1 {
+  color: #ff6b00;
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+}
+
+div h2 {
+  color: #fff5e1;
+  font-weight: 700;
+  font-family: 'Roboto', sans-serif;
+  font-size: 24px;
+  line-height: 32px;
+}
+
+div p {
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+a:link {
+  color: white;
+}
+
+a:visited {
+  color: blue;
+}
+
+a:hover {
+  color: brown;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  background-color: #3c3a39;
+  align-items: center;
+  align-content: space-between;
+  width: 422px;
```

Comment:
- [ ] please set this width to 100% so as to make it responsive, also justify-content: space-between